vue-components/src/components/Button.vue

```diff
@@ -23,9 +23,9 @@ export default Vue.extend( {
 	name: 'Button',
 	props: {
 		/**
-		 * The type of the button
+		 * The flavor of the button
```

Comment:
Maybe using the word "typology" or "style"  instead of flavor in the description could make it easier to understand the prop?

Reply:
`style` is also an attribute, so I wouldn't name it like that.
I don't know about `typology`. Is it more to the point than `flavor`? What do others think?

Reply:
Personally I prefer `flavor`, as it's more snazzy than `typology`. Could live with `variant` as well. In [reactstrap](https://reactstrap.github.io/components/buttons/) they call it `color` but I don't like that...

Reply:
Hey! 👋  I wasn't questioning the use of `flavor`. I just wanted to make sure that the description of the prop is clear enough for those not familiar with the jargon. 
As a side note, the `type` prop is also used with the Message component. I believe we can keep `type` there, since `flavor` might not work well. Let me know if you agree.

Reply:
> Hey! wave I wasn't questioning the use of `flavor`. I just wanted to make sure that the description of the prop is clear enough for those not familiar with the jargon.

Ok cool, can you please make a suggestion from the UI here on github to change it to `typology` and I'll accept it.

> As a side note, the `type` prop is also used with the Message component. I believe we can keep `type` there, since `flavor` might not work well in that case. Let me know if you agree.

It's fine to leave the `type` prop in `Message` because that component's element is a `<div>`, which cannot have a native attribute `type`